numpy/core/src/multiarray/common_dtype.c

```diff
@@ -41,7 +41,7 @@
  * @param dtype2 Second DType class.
  * @return The common DType or NULL with an error set
  */
-NPY_NO_EXPORT NPY_INLINE PyArray_DTypeMeta *
+NPY_NO_EXPORT PyArray_DTypeMeta *
```

Comment:
What is the problem with `NPY_INLINE` here?

Reply:
Since it is no export and inline, clang-cl only lets it be used in the same translation unit. It is later linked across object files which makes clang-cl error.